### DISCUSSION TITLE
Have a problem of generating a 2D Voronoi model

### DISCUSSION THREAD

**Discussion by Chopstickyujie** [neperfepx/neper#798] [Q&A] [answered]
I am a green hand，and I want to generate a 2D model  in this picture(A circle that has been partially cut off) ，the 2D model contains 240 grains. Please help me.
![微信图片_20231229110536](https://github.com/neperfepx/neper/assets/117266409/0e30b246-a5b6-40c7-b982-5686001634e2)

**Comment by bright-men** [comment]
Have you found a way to generate this model？

**Reply by Chopstickyujie** [reply, marked as answer]
I tried this command later, hoping it would help you
![2024122204011](https://github.com/neperfepx/neper/assets/117266409/9d6b1263-46d9-4a46-88e9-74fe33f40dfd)

**Reply by bright-men** [reply]
Many thanks!
